vercel.json

```diff
@@ -76,6 +76,198 @@
     {
       "source": "/inspector",
       "destination": "/docs/implementation/avo-inspector-sdk-reference"
+    },
+    {
+      "source": "/docs/best-practices/avo-functions-alongside-existing-tracking",
+      "destination": "/docs/implementation/start-using-avo-functions"
```

Comment:
Should it actually include `/docs`? Does not work in the vercel build at least 🤔